app/components/ui/molecules/DashboardListItem.js

```diff
@@ -78,31 +80,110 @@ const DateBox = styled(Flex)`
     flex: 0 0 120px;
   `};
 `
-
 const AbsoluteDate = styled.time`
   ${media.tabletPortraitUp`
     font-size: ${th('fontSizeHeading6')};
   `};
 `
 
-const DashboardListItem = ({ manuscript }) => {
-  const date = new Date(manuscript.created)
-  return (
-    <Root py={3}>
-      <TitleBox
-        color={mapColor(manuscript.clientStatus)}
-        data-test-id="title"
-        mb={3}
-      >
-        {manuscript.meta.title || '(Untitled)'}
-      </TitleBox>
-      <ManuscriptStatus statusCode={manuscript.clientStatus} />
-      <DateBox>
-        <time>{dashboardDateText(date)}</time>
-        <AbsoluteDate>{format(date, 'ddd D MMM YYYY')}</AbsoluteDate>
-      </DateBox>
-    </Root>
-  )
+const TrashIcon = props => (
+  <Icon
+    iconName="Trash"
+    overrideName="@pubsweet-pending.PeoplePicker.PersonPod.Remove"
+    {...props}
+  />
+)
+
+const StyledRemoveIcon = styled(TrashIcon)`
+  margin-left: 24px;
+  fill: ${th('colorTextSecondary')};
+`
+
+const TrashButton = styled(Box)`
+  width: 5%;
+`
+
+const DashboardLink = styled(Link)`
+  color: ${th('colorText')};
+  text-decoration: none;
+  width: 95%
+  :hover {
+    color: ${th('colorPrimary')};
+  }
+`
+const DashboardLinkFake = styled.div`
+  width: 95%;
+`
+
+const ItemContent = styled(Flex)`
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+class DashboardListItem extends React.Component {
+  state = {
+    isDeleted: false,
+  }
+
+  renderItemContent = () => {
+    const { meta, clientStatus, created } = this.props.manuscript
+    const date = new Date(created)
+    return (
+      <ItemContent>
+        <TitleBox color={mapColor(clientStatus)} data-test-id="title" mb={3}>
+          {meta.title || '(Untitled)'}
+        </TitleBox>
+        <ManuscriptStatus statusCode={clientStatus} />
+        <DateBox>
+          <time>{dashboardDateText(date)}</time>
+          <AbsoluteDate>{format(date, 'ddd D MMM YYYY')}</AbsoluteDate>
+        </DateBox>
+      </ItemContent>
+    )
+  }
+
+  onClickHandler = deleteManuscript => {
+    deleteManuscript({
+      variables: { id: this.props.manuscript.id },
+    }).then(() => {
+      this.setState({ isDeleted: true })
+    })
+  }
+  render() {
+    const { manuscript } = this.props
+
+    return (
+      !this.state.isDeleted && (
```

Comment:
I've not had to delete using a mutation in the past, and surprisingly, it's still a question which [doesn't have a good answer](https://github.com/apollographql/apollo-client/issues/899)

I think @Cooryd's first option, where we manually refresh the manuscripts list and refetch is the best option, since [it's not advised to mess around with the Apollo cache at the moment](https://github.com/apollographql/apollo-client/issues/621).

Reply:
```
<Mutation refetchQueries={ [{ query: ALL_MANUSCRIPTS }]} /> 
```

That should do the trick. Protects us from damaging the cache but should perform a refresh in the same request as the mutation.